persist-cli-tests/src/test/java/io/ballerina/persist/tools/ToolingInitTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 *  Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org).
```

Reply:
Fixed in [5a7297f](https://github.com/ballerina-platform/persist-tools/pull/336/commits/5a7297f297e0a3e36d42d91d70bb4bbf78856fb1)